composer.json

```diff
@@ -4,11 +4,15 @@
     "type": "project",
     "description": "Symfony Demo Application",
     "autoload": {
-        "psr-4": { "": "src/" },
+        "psr-4": {
+            "AppBundle\\": "src/AppBundle/",
+            "CodeExplorerBundle\\": "src/CodeExplorerBundle/"
+        },
         "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
     },
     "autoload-dev": {
-        "psr-4": { "Tests\\": "tests/" }
+        "psr-4": { "Tests\\": "tests/" },
+        "files": [ "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php" ]
```

Comment:
This is more as to ask for clarifications. I seem to be unable to use the `dump` function when the app is not bootstrapped sometimes so I took the habit to add a shortcut to make a FQCN call to avoid that problem, but this still looks weird as I would expect to be able to use `dump` at anytime.